internal/web/v1/api.go

```diff
@@ -1,11 +1,10 @@
 package v1
 
 import (
+	"github.com/eurofurence/reg-room-service/internal/web/v1/health"
```

Comment:
Can we make use of `gci`? 
Would propose to:
* remove go-import
* Add gci
in the golangci.yaml.

And add a section config:
https://golangci-lint.run/usage/linters/#gci

Then we make sure to have imports properly sorted